app/views/spree/shared/_braintree_hosted_fields.html.erb

```diff
@@ -23,7 +23,7 @@
     <%= label_tag "card_code#{payment_method.id}", Spree::CreditCard.human_attribute_name(:card_code), class: "required" %>
     <div class="input" id="card_code<%= payment_method.id %>"></div>
 
-    <a href="/content/cvv" class="info cvvLink" target="_blank">
+    <a href="/solidus_braintree/content/cvv" class="info cvvLink" target="_blank">
```

Comment:
@elia @kennyadsl The CVV popup was removed from StarterFrontend in https://github.com/solidusio/solidus_starter_frontend/pull/10. However, SolidusBraintree has a link to the popup. To fix the issue, I copied the CVV page from SolidusFrontend. At the moment, it's rudimentary: the link just opens an unformatted page containing the CVV information

Moving forward, we have several options:

1. Restore the CVV popup in StarterFrontend.
2. Retain the CVV popup here and let each extension render its own popup.
3. Remove the CVV link and popup altogether.

Your thoughts?

Reply:
Okay, let's remove the CVV link :+1: